Automattic/wp-calypso
# [WIP] Seller-experience: Add new step to editor welcome tour

#### Changes proposed in this Pull Request

## Closing PR in favor of [61771](https://github.com/Automattic/wp-calypso/pull/61771/files)

* Add new 'Payment block' step to editor welcome tour

#### Testing instructions



To see the new step:

* [Sync](https://github.com/Automattic/wp-calypso/tree/trunk/apps) calypso's `editing-toolkit` app to your sandbox
* Sandbox a new site url
* Create this new site using the simple seller flow
* Its editor should open on the end of the flow `https://wordpress.com/site-editor/<your-site>`
* There should be a "Payments block" step (the second-to-last step)
* If you follow any other flow, you shouldn't get the Payments step.

![image](https://user-images.githubusercontent.com/3801502/156362884-81dfa06d-2112-45e6-a3e2-18fcd40a9200.png)




Closes #61466
